app/tests/spec/models/auth_brokers/oauth-redirect.js

```diff
@@ -216,7 +495,7 @@ define(function (require, exports, module) {
         return broker.afterCompleteResetPassword(account)
           .then((behavior) => {
             assert.isFalse(broker.finishOAuthFlow.called);
-            assert.equal(behavior.type, 'null');
+            assert.equal(behavior.type, 'navigate');
```

Comment:
This change needs to be reverted and happens because [state is set on the relier](https://github.com/mozilla/fxa-content-server/pull/6960/files#diff-44627fb2864aedaf909b5540479e05e3R76). Whenever the user verifies in a different tab, the relier has no state. Can you do a `relier.unset('state')` right after the `account.set` above, and leave a comment about why it's being unset?